types/office-runtime/index.d.ts

```diff
@@ -166,71 +166,73 @@ declare namespace OfficeRuntime {
-   */
-  interface AuthOptions {
     /**
-     * Allows Office to get an access token silently or through interactive consent, if one is required.
-     * If set to false, Office will silently try to get an access token. If it fails to do so, Office will return a descriptive error.
-     * If set to true, Office will show an interactive consent UI after it fails to silently get an access token.
-     * The prompt will only allow consent to the AAD profile scope, not to any Microsoft Graph scopes.
-     */
-    allowConsentPrompt?: boolean;
-    /**
-     * Allows Office to get an access token silently provided consent is present or show interactive UI to sign in the user.
-     * If set to false, office will silently try to get an access token. If it fails to do so, Office will return a descriptive error.
-     * If set to true, Office will show an interactive sign-in UI after it fails to silently get an access token.
-     */
-    allowSignInPrompt?: boolean;
-    /**
-     * Deprecated:
-     * Prompts the user to add their Office account (or to switch to it, if it is already added).
-     */
-    forceAddAccount?: boolean;
-    /**
-     * Deprecated:
-     * Causes Office to display the add-in consent experience. Useful if the add-in's Azure permissions have changed or if the user's consent has
-     * been revoked.
-     */
-    forceConsent?: boolean;
-    /**
-     * Causes Office to prompt the user to provide the additional factor when the tenancy being targeted by Microsoft Graph requires multifactor
-     * authentication. The string value identifies the type of additional factor that is required. In most cases, you won't know at development
-     * time whether the user's tenant requires an additional factor or what the string should be. So this option would be used in a "second try"
-     * call of getAccessToken after Microsoft Graph has sent an error requesting the additional factor and containing the string that should
-     * be used with the authChallenge option.
-     */
-    authChallenge?: string;
-    /**
-     * A user-defined item of any type that is returned, unchanged, in the asyncContext property of the AsyncResult object that is passed to a callback.
-     */
-    asyncContext?: any;
-    /**
-     * Causes Office to return descriptive error when the add-in wants to access MS Graph and the user/admin has not granted consent to MS Graph scopes.
-     * Office only supports consent to graph scopes when the add-in has been deployed by a tenant admin. This information will not be available during development.
-     * Setting this option to true will allow Office to inform your add-in beforehand if MS graph access will fail by returning back a descriptive error.
-     */
-    forMSGraphAccess?: boolean;
+     * Provides options for the user experience when Office obtains an access token to the add-in from AAD v. 2.0 with the `getAccessToken` method.
```

Comment:
```suggestion
     * Provides options for the user experience when Office obtains an access token to the add-in from AAD v. 2.0 with the `getAccessTokenAsync` method.
```

Reply:
@ElizabethSamuel-MSFT In the OfficeRuntime namespace, the method is getAccessToken, not getAccessTokenAsync as it is in the Office namespace.

Reply:
Ah, I see. Thanks.